Patches/Core/ThingDefs_Races/Races_Mechanoid.xml

```diff
@@ -339,6 +356,10 @@
 		</value>
 	</Operation>
 
+	<Operation Class="PatchOperationRemove">
+		<xpath>Defs/ThingDef[defName="Mech_Lancer"]/race/baseHealthScale</xpath>
```

Comment:
I wouldn't remove this--edit maybe, but not remove. Looking at the lancer's parent `BaseMechanoidWalker`, it doesn't have a specific `baseHealthScale`, nor does _its_ parent. So, that makes it unclear where it's getting its `baseHealthScale` stat from and less liable to get changed unexpectedly or unintentionally.